### PR TITLE
Adds HOWTO update the website

### PR DESCRIPTION
**Pull request by lukaszlenart**
A simple HOWTO update the website.

### CODE REVIEW

File: source/updating-website.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+also any Pull Request should be opened against this branch as well.
+
+We are using Jekyll and mix of Markdown pages with raw Html pages, where Markdown is a preferred option.
+
+> WARN: do not manually change the `asf-site` or `ast-staging` branches, they will be automatically updated
+> by CI server. 
+
+## Applying a change
+
+If you are a contributor, and the change is small you can push it directly to the `master` branch. In any other case
```
Comment (rgielen) [inline source/updating-website.md:32]: Are direct commits to master really picked up to be deployed without staging? The small URL fix in primer.md for WW-5059 was directly pushed to master and did not make it to the production site.

File: source/updating-website.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+please open a Pull Request. The Pull Request will be automatically build and deployed to the [staging site](https://struts.staged.apache.org/).
+
```
Comment (rgielen) [inline source/updating-website.md:34]: Shouldn't we refer to http://s.apache.org/asfyaml and BuildBot to help understand how the actual build is performed?

File: source/updating-website.md (hunk 1 above)
Comment (lukaszlenart) [reply source/updating-website.md:32]: They should be, if not we can ask INFRA to check what's wrong.

File: source/updating-website.md (hunk 2 above)
Comment (lukaszlenart) [reply source/updating-website.md:34]: It is, in the `The setup` section

Comment (rgielen) [reply source/updating-website.md:34]: Ah, thanks - oversaw this!

**Pull request merged**